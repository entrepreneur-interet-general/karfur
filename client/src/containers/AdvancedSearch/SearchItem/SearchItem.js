import React, { Component } from 'react';
import { withTranslation } from 'react-i18next';
import { Dropdown, DropdownToggle, DropdownMenu } from 'reactstrap';
import ReactDependentScript from 'react-dependent-script';
import Autocomplete from 'react-google-autocomplete';

import FSearchBtn from '../../../components/FigmaUI/FSearchBtn/FSearchBtn';
import EVAIcon from '../../../components/UI/EVAIcon/EVAIcon';

import "./SearchItem.scss";
// import variables from 'scss/colors.scss';

export class SearchItem extends Component {
  constructor(props) {
    super(props);
    this.selectParam = this.onPlaceSelected.bind(this); //Placé ici pour être reconnu par les tests unitaires
  }
  state = {
    dropdownOpen: false,
    isMounted: false,
    ville: "",
  }

  componentDidMount(){
    this.setState({isMounted: true})
  }

  onPlaceSelected(place){
    this.setState({ville: place.formatted_address });
    this.props.selectParam(this.props.keyValue, place);
  }

  toggle = () => this.setState(prevState=>{return {dropdownOpen:!prevState.dropdownOpen}})
  handleChange = (e) => this.setState({ [e.currentTarget.id]: e.target.value });
  initializeVille = () => this.setState({ville: "" });

  selectOption = subi => {
    this.props.selectParam(this.props.keyValue, subi);
    this.toggle();
  }

  render() {
    const {t, item, keyValue} = this.props;
    const {dropdownOpen, isMounted, ville}=this.state;

    return (
      <div className="search-col">
        <span className="mr-10">{t("SearchItem." + item.title, item.title)}</span>
        {item.queryName === 'localisation' ? 
          isMounted && 
            <ReactDependentScript
              loadingComponent={<div>Chargement de Google Maps...</div>}
              scripts={["https://maps.googleapis.com/maps/api/js?key=" + process.env.REACT_APP_GOOGLE_API_KEY + "&v=3.exp&libraries=places&language=fr&region=FR"]}
            > 
              <div className="position-relative">
                <Autocomplete
                  className={"search-btn in-header search-autocomplete " + (item.active ? "active" : "")}
                  placeholder={item.placeholder}
                  id="ville"
                  value={ville}
                  onChange={this.handleChange}
                  onPlaceSelected={this.onPlaceSelected}
                  types={['(regions)']}
                  componentRestrictions={{country: "fr"}}
                />
                {item.active && 
                  <EVAIcon name="close-circle" size="xlarge" className="close-icon" onClick={e => {e.stopPropagation(); this.props.desactiver(keyValue); this.initializeVille()}} />}
              </div>
            </ReactDependentScript> :
          <Dropdown isOpen={dropdownOpen} toggle={this.toggle} className="display-inline-block">
            <DropdownToggle
              caret
              tag="div"
              data-toggle="dropdown"
              aria-expanded={dropdownOpen}
              className={"search-btn in-header " + (item.short && item.active ? ("bg-" + item.short.split(" ").join("-")) : "") + (!item.short && item.active ? "active" : "")}
            >
              {item.value ? t("Tags." + item.value, item.value) : t("Tags." + item.placeholder, item.placeholder)}
              {item.active && 
                <EVAIcon name="close-circle" size="xlarge" className="close-icon" onClick={e => {e.stopPropagation(); this.props.desactiver(keyValue)}} />}
            </DropdownToggle>
            <DropdownMenu>
              <div className="options-wrapper">
                {item.children.map((subi, idx) => {
                  return(
                  <FSearchBtn 
                    key={idx} 
                    onClick={()=> this.selectOption(subi)}
                    className={"search-options color" + (subi.short ? "": " full")}
                    color={(subi.short || '').replace(/ /g, "-")} 
                  >
                    {t("Tags." + subi.name, subi.name)}
                  </FSearchBtn>
                )})}
              </div>
            </DropdownMenu>
          </Dropdown>}
        {item.title2 &&
          <span className="ml-10">{t("SearchItem." + item.title2, item.title2)}</span>}
      </div>
    )
  }
}

export default withTranslation()(SearchItem);

