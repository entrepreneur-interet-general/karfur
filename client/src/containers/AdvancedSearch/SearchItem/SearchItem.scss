@import 'src/scss/colors.scss';

$placeholderColor: $blanc;

.search-col{
  padding: 10px;
  display: flex;
  justify-content: center;
  align-items: center;

  .dropdown-menu.show{
    display: flex;
    box-shadow: 0px 4px 7px rgba(0, 0, 0, 0.2);
    background: $noir;
    border-radius: 12px;
    padding: 8px;
    max-width: 770px;
    z-index: 3;
    // flex-wrap: wrap;
  }

  .options-wrapper{
    margin-top:-10px;
    margin-bottom: -15px;
    max-height: 50vh;
    overflow: scroll;
    -ms-overflow-style: none;
    overflow: -moz-scrollbars-none;
    &::-webkit-scrollbar { width: 0 !important }
  }
  .search-options{
    white-space: nowrap;
    overflow: visible;
    margin-top:10px;

    &:not(:last-child){
      margin-right: 8px;
    }
  }
  .display-inline-block{
    display:inline-block;
  }
  .texte-blanc{
    color: #FFFFFF;
  }

  .search-dropdown{
    background: $gris;
    border-radius: 6px;
    padding: 20px;
  }

  .search-autocomplete{
    width: 150px;
    text-align: center;
    &::-webkit-input-placeholder {
      color: $placeholderColor;
      text-align: center;
    }
    &:-moz-placeholder {
      color: $placeholderColor;
      opacity: 1;
      text-align: center;
    }
    &::-moz-placeholder {
      color: $placeholderColor;
      opacity: 1;
      text-align: center;
    }
    &:-ms-input-placeholder {
      color: $placeholderColor;
      text-align: center;
    }
    &::-ms-input-placeholder {
      color: $placeholderColor;
      text-align: center;
    }
    &::placeholder {
      color: $placeholderColor;
      text-align: center;
    }
  }
}