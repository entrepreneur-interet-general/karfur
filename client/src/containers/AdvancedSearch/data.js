import {filtres} from "../Dispositif/data";

const initial_data = [
  // {
  //   title:'Je suis',
  //   value: 'réfugié ou accompagnant',
  //   query: 'réfugié',
  //   queryName: 'audience',
  //   children:[
  //     {
  //       name: 'réfugié ou accompagnant',
  //       query:'réfugié',
  //     },
  //     {
  //       name: 'une organisation',
  //       query:'associations',
  //     }
  //   ]
  // },
  {
    title:'Je cherche à',
    value: 'apprendre le français',
    query: 'Apprendre le français',
    queryName: 'tags.name',
    children: filtres.tags
  },
  {
    title:'J\'habite à',
    value: 'Nantes',
    queryName: 'localisation',
    children:[
      {
        name: 'Paris',
      },
      {
        name: 'Lyon',
      },
      {
        name: 'Besançon',
      },
      {
        name: 'Nîmes',
      },
      {
        name: 'Arles'
      }
    ]
  },
  {
    title:'J\'ai',
    value: 'entre 18 et 25 ans',
    queryName: 'audienceAge',
    children:[
      {
        name: "moins de 18 ans",
        bottomValue:0,
        topValue:18
      },
      {
        name: "entre 18 et 25 ans",
        bottomValue:18,
        topValue:25
      },
      {
        name: "entre 25 et 56 ans",
        bottomValue:25,
        topValue:56
      },
      {
        name: "56 ans et plus",
        bottomValue:56,
        topValue:120
      }
    ]
  },
  {
    title:'Je parle',
    value: 'un peu',
    title2:'français.',
    queryName: 'niveauFrancais',
    append: 'Quel est mon niveau ?',
    children:[
      {
        name: "pas du tout",
        query: "Débutant",
      },
      {
        name: "un peu",
        query: "Débutant",
      },
      {
        name: "moyennement",
        query: "Intermédiaire",
      },
      {
        name: "bien",
        query: "Intermédiaire",
      },
      {
        name: "très bien",
        query: "Avancé",
      }
    ]
  }
];

export {initial_data};