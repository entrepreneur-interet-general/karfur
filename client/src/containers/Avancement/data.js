const diffData={
  all: {
    title:'Avancement par langue',
    headers:['Type de contenu', 'Titre du contenu', 'Mots traduits', 'Avancement', 'En attente depuis', '']
  },
  traducteur: {
    title:'Avancement de ',
    headers:['Type de contenu', 'Titre du contenu', 'Mots traduits', 'Avancement', 'En attente depuis', '']
  },
  expert: {
    title:'Avancement de ',
    headers:['Type de contenu', 'Titre du contenu', 'Nombre de mots', 'Traducteurs', 'En attente depuis', '']
  }
};

export {diffData};