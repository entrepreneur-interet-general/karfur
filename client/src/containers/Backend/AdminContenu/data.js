export const table_contenu={
  title: 'Contenu',
  headers: [{
    name: 'Type',
    order: 'typeContenu',
    hideOnPhone: false
   },{
    name: 'Titre',
    order: 'titreCourt',
    hideOnPhone: false
   },{
    name: 'Structure',
    order: 'structure',
    hideOnPhone: false
   },{
    name: 'Depuis',
    order: 'updatedAt',
    hideOnPhone: true
   },{
    name: 'Statut',
    order: 'status',
    hideOnPhone: true
   },{
    name: 'Responsable',
    order: 'responsable',
    hideOnPhone: false
   },{
    name: 'Action interne',
    order: 'internal_action',
    hideOnPhone: false
   },{
    name: 'Variantes',
    order: 'children.length',
    hideOnPhone: false
   },{
    name: 'Suppr',
    hideOnPhone: false
   },{
    name: 'Voir',
    hideOnPhone: false
   },{
    name: 'Publier',
    hideOnPhone: false
   }
  ],
}

export const status_mapping=[
  {
    name: "Actif",
    color: ""
  }, {
    name: "Accepté structure",
    color: "attention"
  }, {
    name: "En attente",
    color: "standby"
  }, {
    name: "En attente admin",
    color: "validation"
  }, {
    name: "En attente non prioritaire",
    color: "focus"
  }, {
    name: "Brouillon",
    color: "attention"
  }, {
    name: "Rejeté structure",
    color: "erreur"
  }, {
    name: "Rejeté admin",
    color: "erreur"
  }, {
    name: "Inactif",
    color: ""
  }, {
    name: "Supprimé",
    color: "erreur"
  }, 
]

export const responsables = ["Hugo", "Simon", "Nour", "Développeur", "Groot", "Starlord"];

export const internal_actions = ["Prêt", "Contact", "Relire", "En attente", "Refaire", "URGENT", "Discuter", "Nouveau"];

export const status_sort_arr = ["En attente admin", "En attente", "En attente non prioritaire", "Brouillon", "Accepté structure", "Actif", "Rejeté structure", "Rejeté admin", "Supprimé", "Inactif" ];