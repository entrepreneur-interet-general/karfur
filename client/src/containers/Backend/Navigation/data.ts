import { SelectedPage } from "./Navigation.component";

export const navigationData: {
  type: SelectedPage;
  iconName: string;
  title: string;
}[] = [
  {
    type: "notifications",
    iconName: "bell",

    title: "Mes notifications",
  },
  {
    type: "favoris",
    iconName: "bookmark",

    title: "Mes favoris",
  },
  {
    type: "contributions",
    iconName: "file-add",

    title: "Mes fiches",
  },
  {
    type: "traductions",
    iconName: "play-circle",

    title: "Mes traductions",
  },
  {
    type: "structure",
    iconName: "briefcase",

    title: "Ma structure",
  },
  {
    type: "profil",
    iconName: "person",

    title: "Mon profil",
  },
  { type: "admin", iconName: "shield", title: "Admin" },
  {
    type: "logout",
    iconName: "log-out",

    title: "DÃ©connexion",
  },
];
