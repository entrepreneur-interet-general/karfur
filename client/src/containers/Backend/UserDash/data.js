import React from 'react';

const steps = [
  {
    content: <div><h3>Bienvenue dans votre espace !</h3><p>Commençons ensemble la visite...</p></div>,
    placement: 'center',
    locale: { 
      skip: 'Passer',
      next: 'Suivant'
    },
    target: 'body',
  },
  {
    title: 'Ma photo',
    content: 'Ici je peux éditer ma photo de profil et changer l\'image que reçoivent de moi les autres traducteurs',
    placement: 'bottom',
    target: '.profile-header-container',
    locale: { 
      skip: 'Passer',
      next: 'Suivant',
      back : 'Précédant'
    },
  },
  {
    title: 'Ma progression',
    content: 'Je peux me définir mes propres objectifs que j\'essaie de tenir tous les jour',
    placement: 'left',
    target: '.my-target-widget',
    locale: { 
      skip: 'Passer',
      next: 'Suivant',
      back : 'Précédant',
      last : 'Terminer'
    },
  }
]

const avancement_langue={
  title: 'Traductions',
  headers: ['Titre', 'Statut', 'Progression', 'Langue', 'Ils rédigent avec moi',''],
  hideOnPhone: [false, false, true, false, true, false]
}

const avancement_data={
  title: 'Commencer à traduire',
  headers: ['Langue', 'Progression', 'Traducteurs mobilisés', ''],
  hideOnPhone: [false,true,true,false],
}

export {steps, avancement_langue, avancement_data}