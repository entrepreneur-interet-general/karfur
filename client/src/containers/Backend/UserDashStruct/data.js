import React from 'react';
import moment from 'moment/min/moment-with-locales';

import EVAIcon from "../../../components/UI/EVAIcon/EVAIcon";

import variables from 'scss/colors.scss';

export const languages = [
  {
    name: 'Anglais',
    code: 'gb',
    avancement: .8,
    nbParticipants:4,
    premiere_image: Math.floor(Math.random() * Math.floor(15))
  },
  {
    name: 'Espagnol',
    code: 'es',
    avancement: .7,
    nbParticipants:3,
    premiere_image: Math.floor(Math.random() * Math.floor(15))
  },
  {
    name: 'Arabe',
    code: 'ma',
    avancement: .55,
    nbParticipants:1,
    premiere_image: Math.floor(Math.random() * Math.floor(15))
  }
]

export const past_translation = [
  {
    name: 'Anglais',
    code: 'gb',
    titre: 'Les 3 petits cochons',
    statut: 'Validée'
  },
  {
    name: 'Anglais',
    code: 'gb',
    titre: 'Blanche neige et les 7 nains',
    statut: 'En attente'
  },
  {
    name: 'Espagnol',
    code: 'es',
    titre: 'Les 3 petits cochons',
    statut: 'En cours'
  },
  {
    name: 'Espagnol',
    code: 'es',
    titre: 'Les chiens aboient quand la caravane passe',
    statut: 'Annulée'
  }
]

export const steps = [
  {
    content: <div><h3>Bienvenue dans votre espace !</h3><p>Commençons ensemble la visite...</p></div>,
    placement: 'center',
    locale: { 
      skip: 'Passer',
      next: 'Suivant'
    },
    target: 'body',
  },
  {
    title: 'Ma photo',
    content: 'Ici je peux éditer ma photo de profil et changer l\'image que reçoivent de moi les autres traducteurs',
    placement: 'bottom',
    target: '.profile-header-container',
    locale: { 
      skip: 'Passer',
      next: 'Suivant',
      back : 'Précédant'
    },
  },
  {
    title: 'Ma progression',
    content: 'Je peux me définir mes propres objectifs que j\'essaie de tenir tous les jour',
    placement: 'left',
    target: '.my-target-widget',
    locale: { 
      skip: 'Passer',
      next: 'Suivant',
      back : 'Précédant',
      last : 'Terminer'
    },
  }
]

export const avancement_actions={
  title: 'Notifications',
  headers: [<EVAIcon name="bell-outline" fill={variables.noir} />, 'Titre', 'Mon rôle', 'Type', 'Depuis', '', ''],
  hideOnPhone: [false, false, true, false, true, false, false]
}

export const fakeNotifs = {
  "_id" : "unidentifiantquelconque",
  "titre" : "Faire son service civique en France",
  "owner": true,
  "action": "questions",
}

export const avancement_contributions={
  title: 'Contributions',
  headers: ['Titre', 'Statut', 'Ils rédigent avec moi',''],
  hideOnPhone: [false, false, true, false]
};

export const avancement_members={
  title: 'Membres',
  headers: ['Nom', 'Rôle', 'Dernière connexion', 'Ajouté le', '', '', ''],
  hideOnPhone: [false, false, true, true, true, true, true]
}

export const fakeContribution = {
  "_id" : "unidentifiantquelconque",
  "titreInformatif" : "Un exemple de dispositif",
  "titreMarque" : "Le nom du dispositif",
  "avancement" : 1,
  "status" : "Brouillon",
  "creatorId" : "unidentifiantquelconque",
}

export const fakeMembre = {
  "_id" : "unidentifiantquelconque",
  "username" : "Simon Karleskind",
  "picture": {
    "imgId" : "5ced0172d97ab7f4c6714879",
    "public_id" : "pictures/zgjpujxzk9kxoqylt93n",
    "secure_url" : "https://res.cloudinary.com/dlmqnnhp6/image/upload/v1559036274/pictures/zgjpujxzk9kxoqylt93n.jpg"
  },
  "role": "Contributeur",
  "last_connected": moment().add(-1, 'month'),
  "created_at": moment().add(-3, 'month'),
  "nb_contenu": 3,
}