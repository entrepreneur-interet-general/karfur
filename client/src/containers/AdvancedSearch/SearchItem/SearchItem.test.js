import React from 'react';
import {SearchItem} from './SearchItem';
import { expect } from 'chai';
import { shallow, mount } from 'enzyme';
import sinon from 'sinon';
import lolex from 'lolex';
import { Dropdown } from 'reactstrap';

import { initial_data } from '../data';
import EVAIcon from '../../../components/UI/EVAIcon/EVAIcon';
import FSearchBtn from '../../../components/FigmaUI/FSearchBtn/FSearchBtn';
import { filtres } from '../../Dispositif/data';

describe('SearchItem', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <SearchItem 
        t={k=>k}
        item={initial_data[0]}
        keyValue={0}
        selectParam = {()=>{}}
        desactiver={()=>{}}
      />
    );
  })
  
  it('renders without crashing', () => {
    expect(wrapper.is('div')).to.equal(true);
    expect(wrapper.find('span').at(0).text()).to.equal("SearchItem.Je cherche à");
  });

  it('calls componentDidMount', () => {
    sinon.spy(SearchItem.prototype, 'componentDidMount');
    shallow(<SearchItem 
      t={k=>k}
      item={initial_data[0]}
      keyValue={0}
      selectParam = {()=>{}}
      desactiver={()=>{}}
    />);
    expect(SearchItem.prototype.componentDidMount).to.have.property('callCount', 1);
    expect(wrapper.state().isMounted).to.be.a('boolean').that.is.true;
    SearchItem.prototype.componentDidMount.restore();
  });

  it('should match its reference snapshot', () => {
    expect(wrapper).to.matchSnapshot()
  });

  it('contains search-col in className', () => {
    expect(wrapper.props().className).to.contain("search-col");
  });

  it('should have an empty users list in state', () => {
    expect(wrapper.state().isMounted).to.be.a('boolean').that.is.false;
  });

  it('does not render children when passed in', () => {
    const wrapper = shallow((
      <SearchItem t={k=>k} item={initial_data[0]} >
        <div className="unique" />
      </SearchItem>
    ));
    expect(wrapper.contains(<div className="unique" />)).to.equal(false);
  });

  it('contains EVAIcon when active', () => {
    wrapper.setProps({item:{...initial_data[0], active: true}})
    expect(wrapper.find(EVAIcon)).to.have.lengthOf(1);
  });

  it('should have a dropdown', () => {
    expect(wrapper.find(Dropdown)).to.have.lengthOf(1);
  });

  it('should have as many options as there are themes', () => {
    expect(wrapper.find(FSearchBtn)).to.have.lengthOf(filtres.tags.length);
  });

  it('mounts without crashing', () => {
    const mounted = mount(<SearchItem t={k=>k} item={initial_data[0]} />).childAt(0);
    expect(mounted.is('div')).to.equal(true);
  });

  //Ne marche pas, le champ de recherche Google ne se charge pas même en attendant longtemps, j'abandonne
  // it('selects a place in searchBar', async(done) => {
  //   sinon.spy(SearchItem.prototype, 'componentDidMount');
  //   const onButtonClick = sinon.spy(SearchItem.prototype, "onPlaceSelected");
  //   const mounted = mount(<SearchItem t={k=>k} item={initial_data[1]} selectParam = {()=>{}} />);
  //   expect(SearchItem.prototype.componentDidMount).to.have.property('callCount', 1);
  //   expect(mounted.state().isMounted).to.be.a('boolean').that.is.true;
  //   console.log(new Date())
  //   // timeout(30000);

  //   await mounted.update()

  //   setTimeout(async function () {
  //     console.log(new Date())
  //     await mounted.update()
  //     console.log(mounted.debug())
  
  //     const btnNode = mounted.find('div.dropdown-toggle');
  //     expect( btnNode ).to.have.lengthOf(1);
  //     btnNode.simulate('click');
  //     const subBtnNode = mounted.find('button.search-btn').at(0);
  //     expect( subBtnNode ).to.have.lengthOf(1);
  //     subBtnNode.simulate('click');
  //     expect(onButtonClick.calledOnce).to.equal(true);
  //     onButtonClick.restore();
  //     clock.restore();
  //     SearchItem.prototype.componentDidMount.restore();
  //     done();
  //   }, 30000);
    
  // }, 30000);
})

