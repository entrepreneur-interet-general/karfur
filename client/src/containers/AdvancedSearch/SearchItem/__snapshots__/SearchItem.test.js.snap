// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchItem should match its reference snapshot 1`] = `
<div
  className="search-col"
>
  <span
    className="mr-10"
  >
    SearchItem.Je cherche à
  </span>
  <Dropdown
    active={false}
    addonType={false}
    className="display-inline-block"
    direction="down"
    inNavbar={false}
    isOpen={false}
    nav={false}
    setActiveFromChild={false}
    toggle={[Function]}
  >
    <DropdownToggle
      aria-expanded={false}
      aria-haspopup={true}
      caret={true}
      className="search-btn in-header "
      color="secondary"
      data-toggle="dropdown"
      tag="div"
    >
      Tags.choisir un thème
    </DropdownToggle>
    <DropdownMenu
      flip={true}
      tag="div"
    >
      <div
        className="options-wrapper"
      >
        <fSearchBtn
          className="search-options color"
          color="Français"
          key="0"
          onClick={[Function]}
        >
          Tags.apprendre le français
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Logement"
          key="1"
          onClick={[Function]}
        >
          Tags.me loger
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Insertion-pro"
          key="2"
          onClick={[Function]}
        >
          Tags.trouver un travail
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Formation-pro"
          key="3"
          onClick={[Function]}
        >
          Tags.apprendre un métier
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Études"
          key="4"
          onClick={[Function]}
        >
          Tags.faire des études
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Mobilité"
          key="5"
          onClick={[Function]}
        >
          Tags.me déplacer
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Bénévolat"
          key="6"
          onClick={[Function]}
        >
          Tags.aider une association
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Rencontre"
          key="7"
          onClick={[Function]}
        >
          Tags.rencontrer des gens
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Loisir"
          key="8"
          onClick={[Function]}
        >
          Tags.occuper mon temps libre
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Culture"
          key="9"
          onClick={[Function]}
        >
          Tags.découvrir la culture
        </fSearchBtn>
        <fSearchBtn
          className="search-options color"
          color="Santé"
          key="10"
          onClick={[Function]}
        >
          Tags.me soigner
        </fSearchBtn>
      </div>
    </DropdownMenu>
  </Dropdown>
</div>
`;
